scikeras/wrappers.py

```diff
@@ -575,7 +577,7 @@ def fit(self, X, y, sample_weight=None):
             X=X, y=y, sample_weight=sample_weight, warm_start=self.warm_start
         )
 
-    def _fit(self, X, y, sample_weight=None, warm_start=False):
+    def _fit(self, X, y, sample_weight=None, warm_start=False, epochs=None):
```

Comment:
Why can't `epochs` be a required parameter?

``` python
def partial_fit(self, X, y):
    return self._fit(X, y, epochs=1, warm_start=True)

def fit(self, X, y):
    return self._fit(X, y, epochs=self.epochs, warm_start=self.warm_start)

def _fit(self, X, y, epochs, warm_start):
    ...
```

This is still callable with `self._fit(X, y, epochs=epochs, warm_start=warm_start)`.

Reply:
It can, I thought this was cleaner, but you seem to think that the alternative is cleaner; I'm happy to change it to what you are suggesting.

Reply:
Changed to a required parameter.